doc/rados/troubleshooting/troubleshooting-osd.rst

```diff
@@ -486,26 +554,47 @@ the internal code (such as passing data across locks into new threads).
 Flapping OSDs
 =============
 
-We recommend using both a public (front-end) network and a cluster (back-end)
-network so that you can better meet the capacity requirements of object
-replication. Another advantage is that you can run a cluster network such that
-it is not connected to the internet, thereby preventing some denial of service
-attacks. When OSDs peer and check heartbeats, they use the cluster (back-end)
+When OSDs peer and check heartbeats, they use the cluster (back-end)
 network when it's available. See `Monitor/OSD Interaction`_ for details.
 
-However, if the cluster (back-end) network fails or develops significant latency
-while the public (front-end) network operates optimally, OSDs currently do not
-handle this situation well. What happens is that OSDs mark each other ``down``
-on the monitor, while marking themselves ``up``. We call this scenario
-'flapping`.
-
-If something is causing OSDs to 'flap' (repeatedly getting marked ``down`` and
-then ``up`` again), you can force the monitors to stop the flapping with::
+We have tradtionally recommended separate *public* (front-end) and *private*
+(cluster / back-end / replication) networks:
+
+#. Segregation of heartbeat and replication / recovery traffic (private)
+   from client and OSD <-> mon traffic (public).  This helps keep one
+   from DoS-ing the other, which could in turn result in a cascading failure.
+
+#. Additional throughput for both public and private traffic.
+
+When common networking technloogies were 100Mb/s and 1Gb/s this separation
```

Comment:
s|1Gb/s|1GB/s,|

(put a comma after "1GB/s")

Reply:
i think "100Mb/s and 1Gb/s" is correct. when it comes to bandwidth, we tend to use bits as the unit. they are also short for "100Mbs and 1Gbps".

Reply:
I know the Google style guide favors commas like this.  I find it stilted but not worth arguing about.

Reply:
tchaikov: I think Zac meant only to suggest adding a comma.   I think `b` => `B` was inadvertent.

From my NSP days I instinctively wanted to write FE, GE, XE but I've found that those terms aren't widely recognized.